lib/timetable-visualisation/PlanPreview.tsx

```diff
@@ -0,0 +1,74 @@
+import { DevelopmentPlan } from "../types/timetable";
+
+import "govuk-frontend/dist/govuk/govuk-frontend.min.css";
+
+type PlanPreviewProps = {
+  plan: DevelopmentPlan;
+};
+
+export const PlanPreview = ({
```

Comment:
I'm being picky here but I'm not sure about the name of this component. It feels weird to call it a "preview" when it's only a preview for the form, it's the final thing for the visualisation tool.

It makes sense to keep the top level component name the same but how about calling this something like `PlanViewer` or something similar?

Reply:
PlanViewer sounds good :)